scripts/text_validator.js

```diff
@@ -1,7 +1,6 @@
-/*global $ goodChars charMatch XRegExp */
-/* exported validateText */
+/*global $ charMatch XRegExp */
```

Comment:
Even if we do decide to go with a regex, it would be better to have the "is valid character" logic be within `character_test.js` rather than exposing a global only to have it be used in this file. Either way we should centralize that logic.